bindings/python/docs/source/supported_types.rst

```diff
@@ -6,6 +6,11 @@ Supported Types
 PyMongoArrow currently supports a small subset of all BSON types.
 Support for additional types will be added in subsequent releases.
 
+.. note:: PyMongoArrow does not currently support extension types with Pandas/NumPy or Arrow. However, they can be used in schemas.
+   This means that ObjectId and Decimal128 are not supported in Pandas DataFrames or Arrow Tables.
+   Instead, the schema type will be converted to a string representation of the type.
```

Comment:
It is binary for object id, and str for decimal128

Reply:
I was considering it from the perspective of there being {fixed_width_binary(12), string, object, object}, so in this intro I would just describe the general case.